website/docs/reference/project-collaboration-mode.md

```diff
@@ -2,52 +2,65 @@
 
-The open collaboration mode is the default in Unleash and is how all projects worked in Unleash before the introduction of collaboration modes.
+Unleash supports three collaboration modes: **open**, **protected**, and **private**. Each mode is designed with different user roles and permissions in mind, catering to various security and collaboration requirements. Let’s explore how the three modes work.
 
-## **Protected** collaboration mode
+## Open collaboration mode
 
-Only admins and project members can submit change requests in a **protected** project. Other users can not submit change requests.
+All users of your Unleash instance can access the project and submit change requests. This is the default collaboration mode.
 
-## **Private** collaboration mode
 
-Private collaboration mode renders the project invisible to all **viewers** who are not project members. This means that **viewers** cannot see the project in the project list, nor can they access the project's features or locate the project anywhere within Unleash.
+## Protected collaboration mode
 
-**Editors**, **admins**, and users with **custom root roles** can still see private projects.
+All users of your Unleash instance can access the project but only admins and project members can submit change requests.
 
-### Change requests
 
-When you change a project's collaboration mode from open to protected, users who do not have the right permissions will lose the ability to create new change requests.
+## Private collaboration mode
 
-However, existing change requests created by these users will not be deleted. Any users with open change requests will still be able to cancel the change request, but they will **not** be able to update the change request.
+All project members can access the project and submit change requests. Outside of project members, only admins, editors, and users with [custom root roles](../how-to/how-to-create-and-assign-custom-root-roles.md) can access the project and submit change requests.
 
-## Project collaboration mode setting
 
-The collaboration mode for a project can be set at the time of creation and modified at any subsequent time, found within the 'Enterprise Settings' section.
+| Collaboration Mode | View project    | Submit change requests                                                                                                        |
+|--------------------|-----------------|-------------------------------------------------------------------------------------------------------------------------------|
+| Open               | All users       | All users                                                                                                                     |
+| Protected          | All users       | Project members, Admins, Editors, and users with [custom root roles](../how-to/how-to-create-and-assign-custom-root-roles.md) |
```

Comment:
I don't think that Editors can submit change requests in the protected mode.

Reply:
Verified that Thomas as an Editor cannot submit change requests